### PR TITLE
[Accuracy diff No.126] Fix accuracy diff for paddle.nn.functional.rnnt_loss API

### PR DESCRIPTION
**Pull request by ooooo-create**
paddle cpu 时不会进行 fused_log_softmax 操作，需要手动对 logits 进行 log_softmax

### CODE REVIEW

**Comment by ooooo-create** [comment]
> cpu相关的处理逻辑在哪里 你能说一下我去看看吗？

这个 pr 描述里面，https://github.com/PaddlePaddle/Paddle/pull/49199
图片
调用的这个库 https://github.com/PaddlePaddle/warp-transducer.git
图片

**Comment by wanghuancoder** [comment]
> > cpu相关的处理逻辑在哪里 你能说一下我去看看吗？
> 
> 这个 pr 描述里面，[PaddlePaddle/Paddle#49199](https://github.com/PaddlePaddle/Paddle/pull/49199) 图片 调用的这个库 https://github.com/PaddlePaddle/warp-transducer.git 图片

我有点儿没绕明白。我们现在是在修复paddle.nn.functional.rnnt_loss的精度问题。然后我看截图，是PR描述中测试CPU性能的时候先调用了log_softmax，然后调用了rnnt_loss。那应该与rnnt_loss的精度问题无关呀？

**Review by wanghuancoder — APPROVED** [review]
